pkg/shared/util/env.go

```diff
@@ -27,3 +29,15 @@ func LookupEnvStringOr(key, defaultValue string) string {
 		return defaultValue
 	}
 }
+
+func LookupEnvInt64Or(key string, defaultValue int) int {
```

Comment:
Function name does not match the type return?

Reply:
made it `LookupEnvIntOr`